samples/editors/date-picker/format/App.razor

```diff
@@ -0,0 +1,22 @@
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor
+
+<div class="container vertical">
```

Comment:
add DIV with options to change DateFormat property

Reply:
pending